media/Config/buildDom.js

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// remove every options on DOM
+const emptyOptionBox = function(isImport) {
+  if (!isImport) {
+    // erase select tag for choosing framework
+    const locaForSelect = document.querySelector('#locaForSelect');
```

Comment:
Just curiosity, what does loca mean? local?

Reply:
loca means location.
locationForSelect is better for understand?

Reply:
What is the meaning of `location` here?

Reply:
`location` means select tag's location in media/Config/index.html.
we want to make select tag in specific area, so we make div tag for select tag.

Reply:
![Cap 2021-10-01 13-43-33-205](https://user-images.githubusercontent.com/77395035/135566461-db424c3d-2f86-4be9-8646-2cde6807a33a.png)

Reply:
Then, just `SelectTag` seems enough. Will it confuse anything?

Reply:
locaForSelect is div tag, so may be confused.
divTagForSelect will be good choice?

Reply:
I think it's better:)

Reply:
sorry, `locaForSelect` is not div tag, it's span tag.
so i changed it `spanTagForSelect'